vulkan/config.gni

```diff
@@ -0,0 +1,12 @@
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+declare_args() {
+  # Whether to include vulkan validation layers, if available.
+  #
+  # Currently these are only supported on Fuchsia, where by default they are
+  # disabled, to enable them pass `--enable-vulkan-validation-layers` to your
+  # gn args.
```

Comment:
Nit: `--enable-vulkan-validation-layers` isn't really a GN arg. That just what the python script that happens to be called gn uses to setup the GN arg.

Reply:
Yup, its kinda confusing since the name of the script is also `gn` and the args for the `gn` python script are not the same as "gn args".